ui/src/shared/constants/tableGraph.js

```diff
@@ -51,6 +52,7 @@ export const calculateTimeColumnWidth = timeFormat => {
   timeFormat = _.replace(timeFormat, 'dddd', 'Wednesday')
   timeFormat = _.replace(timeFormat, 'A', 'AM')
   timeFormat = _.replace(timeFormat, 'h', '00')
+  timeFormat = _.replace(timeFormat, 'X', '1522286058')
```

Comment:
oh clever! But a potential clever way to do this is create a new date and format it using moment and that will give you the actual length of some data. granted their could be outliers, but it takes out some of the need for hardcoding these.